scripts/bot/run-bot.js

```diff
@@ -4,8 +4,9 @@ Usage:
     ./run-bot.js [options]
 
 Options:
-    -h --host=<host>  Hubs host [default: localhost:8080]
-    -r --room=<room>  Room id [default: 234234].
+    -u --url=<url>    Url [default: null].
```

Comment:
``default: null]` will make `options["--url"]` return the string "null", so that 
```const baseUrl = options["--url"] || `https://${options["--host"]}/hub.html`;```
doesn't do the right thing.

Reply:
Oh, good catch. Thanks!